### DISCUSSION TITLE
Queue / sandbox worker without redis

### DISCUSSION THREAD

**Discussion by RichardWright** [taskforcesh/bullmq#2412] [Q&A]
Hi bullmq maintainers 

I’m using bullmq to compress something in another process to keep our event loop from being affected by this compression. As this is intended for running within the same container, does this need a redis instance for the queue? Would initialising ioredis without settings be an alternative to keep it in memory?

**Comment by manast** [comment]
No, you need a Redis instance in order to use BullMQ. In you case, if you do not need to scale, you could just use NodeJS ```worker_threads``` (https://nodejs.org/api/worker_threads.html) for example.
